src/jit/IntMath64Inl.h

```diff
@@ -344,16 +344,27 @@ static void emitUnary(sljit_compiler* compiler, Instruction* instr)
     sljit_emit_op1(compiler, opcode, args[1].arg, args[1].argw, args[0].arg, args[0].argw);
 }
 
+void emitSelect128(sljit_compiler*, Instruction*, sljit_s32);
+
 void emitSelect(sljit_compiler* compiler, Instruction* instr, sljit_s32 type)
 {
     Operand* operands = instr->operands();
     assert(instr->opcode() == ByteCode::SelectOpcode && instr->paramCount() == 3);
+    size_t valueSize = reinterpret_cast<Select*>(instr->byteCode())->valueSize();
 
     if (false) {
```

Comment:
This is changed. Please rebase the patch.

Reply:
Done.